pad/hori/Fighting Stick α/config.json

```diff
@@ -0,0 +1,94 @@
+{
+    "name": "Hori Fighting Stick α for PlayStation 5",
+    "functions": {
+        "analog_sticks": null,
+        "digital_sticks": null,
+        "action_buttons": [
+            {
+                "name": "Square",
+                "analog": false
+            },
+            {
+                "name": "Circle",
+                "analog": false
+            },
+            {
+                "name": "Cross",
+                "analog": false
+            },
+            {
+                "name": "Triangle",
+                "analog": false
+            },
+            {
+                "name": "L1",
+                "analog": false
+            },
+            {
+                "name": "R1",
+                "analog": false
+            },
+            {
+                "name": "L2",
+                "analog": false
+            },
+            {
+                "name": "R2",
+                "analog": false
+            },
+            {
+                "name": "L3",
+                "analog": false
+            },
+            {
+                "name": "R3",
+                "analog": false
+            }
+        ],
+        "menu_buttons": ["Options"],
+        "system_buttons": ["Create", "PS Button"],
+        "other_buttons": ["Function", "Tournament Lock"],
+        "trackpads": [ 
+            {
+            "name": "Trackpad",
+            "clickable": true
+            }
+        ],
+        "touch_screens": null,
+        "directional_buttons": [
+            {
+                "set": [
+                    "Up",
+                    "Down",
+                    "Left",
+                    "Right"
+                ],
+                "socd_prevention": {
+                    "hardware": true,
+                    "software": null
+                }
+            }
+        ],
+        "gyroscope": false,
+        "accelerometer": false,
+        "ir_reciever": false,
+        "extension_ports": 0,
+        "headset_port": true,
+        "speaker": false,
+        "microphone": false,
+        "cable": {
+            "interface": "USB-A",
+            "removable": false
+        },
+        "bluetooth": false,
+        "2_4ghz": false,
+        "rumble": null,
+        "digital_to_analog_conversion": null,
```

Comment:
Does the stick have the capability to input Left Stick / Right Stick commands using the Digital Stick? If so then the Digital-to-Analog Conversion section should be filled in appropriately (See Keybox for example, https://github.com/Wolfy76700/ControllerDatabase/wiki/Controller-Definition#autogenerated_heading_15 for details)